clients/unshaded/src/main/java/tachyon/client/file/FileSystemContext.java

```diff
@@ -72,4 +72,8 @@ public synchronized void reset() {
     mFileSystemMasterClientPool =
         new FileSystemMasterClientPool(ClientContext.getMasterAddress());
   }
+
+  public static FileSystemContext get() {
```

Comment:
Is there any way to enforce callers use `FileSystemContext.get()` instead of `FileSystemContext.INSTANCE`?